pkg/cover/canonicalizer.go

```diff
@@ -132,3 +142,17 @@ func convertModulePCs(moduleKeys []uint32, conversionHash map[uint32]*canonicali
 		}
 	}
 }
+
+func convertSignals(moduleKeys []uint32, conversionHash map[uint32]*canonicalizerModule, sign signal.Serial) {
+	for idx, elem := range sign.Elems {
```

Comment:
FWIW also note that not all signal are PCs, e.g. look at

https://github.com/google/syzkaller/blob/8d5c7541d29fecb0d8adc7f44caedd9abe9c9dca/pkg/ipc/ipc.go#L304-L307

Reply:
Oh interesting. Thanks for letting me know; I'll look into accommodating this

Reply:
I've disabled canonicalization when FlagSignal=false, as in this case no coverage is returned and returned signals are FallbackSignals.